src/test/controllers/survey.controller.test.js

```diff
@@ -23,6 +24,9 @@ describe('Test Survey API', function () {
         form: menteeData.surveyA001_1,
         resolveWithFullResponse: true,
         json: true,
+        headers: {
+          access_token: userData.USER_A_DATA.access_token,
+        },
```

Comment:
Maybe this part didn't need any session.
So when using JWT, can't we divide some non-logined feature and logined feature?